.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+name: Continuous Integration
+on: [push]
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php-version: [ '7.2', '7.3', '7.4', '8.0', '8.1' ]
```

```suggestion
        php-version: ['7.4', '8.0', '8.1' ]
```

if we drop `< php 7.4`